Using git as an Alternative to Robert's Rules.md

```diff
@@ -0,0 +1,68 @@
+
+* How order is kept.
+  * Who gets to speak when?
+  * What is required for a decision to become final?
+* How to make and keep an agenda
+  * Who gets to bring up what topics when?
+
+## Glossary
+
+### Defining
+Things which are immutable during the course of an invocation of the rules.
+
+* chair: The presiding officer of the meeting, generally the chapter President.
+
+### Protocol
+Things which are quantitative, logical, and asynchronous:
+
+* ballot: Voting in writing can take place if the issue being discussed is a sensitive matter and the chair or
+members wish the voting to be secret.
+* motion: The method used to place an issue, question, or decision in front of the members so it can be
+discussed and voted upon. Discussion on an issue to be decided is out of order unless an appropriate
+motion has been made.
+* second: Most motions need a second (that is, another member must agree that the question or issue
+should be discussed). This keeps items that are of interest to only one member from taking up time during
+a meeting.
+
+### Relational
+Things which are conversational, chronological:
+
+* floor: A member is considered to “have the floor” when the chair calls upon him or her to speak. In most
+cases, it is improper to interrupt someone who currently has the floor.
+* order: Comments or discussion presented by a member are “in order” if the chair has properly
+recognized the member and the comments made pertain to the issue being discussed. Likewise, the
+member is “out of order” if either of these two conditions is not met.
+* quorum: The number of members needed to be present to conduct business legally. This number is set
+in your chapter’s bylaws.
+
+### Irrelevant
+Things which may fit into one of the categories above but which are obsoleted by this protocol.
+
+* voice vote: The chair requests that members indicate their preference on the issue by responding “yes”
+or “no” when asked to do so. The chair decides, based on the voices, whether the “yes” or “no” votes
+were in the majority. The chair may ask for a show of hands if it is unclear by voice vote how members
+voted. 
+
+# Proposal to replace Robert's Rules
+
+* ballot: A ballot is a pull request. Pull requests may be opened at any time by any member of the board or organization, similar to how Robert's Rules states that attendees may bring up business not on the agenda.
+* motion: A motion act of opening a pull request.
+* second: A second is a quorum of approvals of the pull request by the board.
+* floor: During board meetings, the floor is the right of a board member to speak. On a pull request, the floor is the right of board members and community members to engage in reasonable helpful discussion about an issue.
```

Comment:
Since this is a proposal, I guess the next step is to add it to our by laws?

Reply:
Yep.